### PR TITLE
Minor Release

### PR DESCRIPTION
**Pull request by tomahawk-pilot**
Changelog:
1) Changes to patient Manage page
2) more filters to shifting
3) new fields in patients
4) changes to the consultation
5)  New user fields
6)bug fixes in manage patient page
7) various other bug fixes
Contributors:

- @tomahawk-pilot 
- @giju 
- @MalayaliRobz 
- @SandeepThomas 
- @bodhish 
- @riginoommen

### CODE REVIEW

File: src/Components/Patient/SampleViewAdmin.tsx [hunk 1]
```diff
@@ -76,23 +76,23 @@ export default function SampleViewAdmin(props: any) {
...
-  const searchByPhone = async (searchValue: string) => {
-    setIsLoading(true);
-    const res = await dispatch(sampleSearch({ limit, offset, phone_number: encodeURI(searchValue) }));
-    if (res && res.data) {
-      setSample(res.data.results);
-      setTotalCount(res.data.count);
-    }
-    setIsLoading(false);
-  }
+  // const searchByPhone = async (searchValue: string) => {
```
Comment (riginoommen) [inline src/Components/Patient/SampleViewAdmin.tsx:87]: Remove the commented code

Comment (dauntlessnomad) [reply src/Components/Patient/SampleViewAdmin.tsx:87]: this will be implemented in the future

**Comment by dauntlessnomad** [comment]
@riginoommen we will merge this at night.

**Comment by riginoommen** [comment]
Yea

**Pull request merged**